jaggery-modules/carbon-metrics/module/scripts/core/core.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var metrics = {};
+(function (metrics) {
+    var log = new Log('carbon.metrics');
+    var MetricManager = org.wso2.carbon.metrics.manager.MetricManager;
+    var WSO2_CONSTANT_METRIC_ENABLED = 'wso2.metrics.enabled';
+    var WSO2_CONSTANT_METRIC_TRACE_LOGS_ENABLED = 'wso2.metrics.trace.logs.enabled';
+    var UUID = Packages.java.util.UUID;
+
+    metrics.init = function (options) {
+        var metricsConfiguration = options.metrics || {};
+        application.put(WSO2_CONSTANT_METRIC_ENABLED, metricsConfiguration.enabled || false);
+        application.put(WSO2_CONSTANT_METRIC_TRACE_LOGS_ENABLED, metricsConfiguration.enableTraceLogs || false);
+    };
+    /**
+     * Checks whether metrics has been enabled by first checking for a global
+     * variable and if not present tries to obtain the enable state from the application
+     * context
+     * @return {Boolean} [description]
+     */
+    var isEnabled = function () {
+        if (typeof WSO2_METRICS_ENABLED === 'undefined') {
+            WSO2_METRICS_ENABLED = application.get(WSO2_CONSTANT_METRIC_ENABLED);
+        }
+        return WSO2_METRICS_ENABLED;
+    };
+    var isTraceLogsEnabled = function(){
+        if(typeof WSO2_METRIC_TRACE_LOGS_ENABLED === 'undefined'){
```

Comment:
Thanks for mentioning , As I know it is a good practice in JavaScript to use **===**  rather than strict comparison.
Source: http://www.w3schools.com/js/js_best_practices.asp